### ISSUE TITLE
Accessibility enhancements

### ISSUE THREAD

**Issue by jsuchal** [slovensko-digital/autogram#722]
## Goal
Fix accessibility bugs in collaboration with users with disabilities and/or NLnet.

## Subtasks
- [ ] 8a. Accessibility testing (partially done)
- [ ] 8b. Fixes/changes based on accessibility testing

## Related issues
- #565 Focus does not automatically move into the window on app start
- #555 Cannot be minimised with Command + M on macOS
- #554 "Show signature details" button cannot be activated with the keyboard
- #552 Button does not report its current state (open/closed)
- #551 Insufficient text colour contrast
- #550 UI elements are not associated with labels
- #503 Poor accessibility in the country list
- #436 Screen reader does not announce a newly opened dialog
- #663 macOS: cannot copy the support contact text
- #373 Windows windows too large at 125% scaling
